pkg/volume/volume.go

```diff
@@ -31,10 +31,15 @@ func Get(vols []string, openebsNS, casType string) error {
 	k, _ := client.NewK8sClient("")
 	// 1. Get a list of required PersistentVolumes
 	var pvList *corev1.PersistentVolumeList
+	var err error
 	if vols == nil {
-		pvList, _ = k.GetPVs(nil, "")
+		pvList, err = k.GetPVs(nil, "")
 	} else {
-		pvList, _ = k.GetPVs(vols, "")
+		pvList, err = k.GetPVs(vols, "")
+	}
+	if err != nil {
```

Comment:
A nil *corev1.PersistentVolumeList shouldn't be passed to the functions, so that each one don't need a nil or empty check.